examples/node-pub

```diff
@@ -18,7 +18,8 @@
 /* jslint node: true */
 'use strict'
 
-const nats = require('nats')
+// change '../' to 'nats' when copying
```

Comment:
Not sure what this means.

Reply:
I made the examples work right from the package when installed as `npx